### PR TITLE
Minor wording edit to post-liberation Poisonwood's spaceport description

### PR DESCRIPTION
**Pull request by ImprovisedUsername**
## Summary
The original wording for Poisonwood's post-liberation spaceport was rather clunky, so I tried to fix it. Slightly longer than it originally was, unfortunately.

Also present are a number of minor edits and corrections to other planet/spaceport descriptions, mostly to do with capitalization, wording, and punctuation.

### CODE REVIEW

File: data/human/campaign events.txt [hunk 1]
```diff
@@ -1789,7 +1789,7 @@ event "liberation of Poisonwood"
...
+		spaceport `	The tavern in the center of the port is named The Parting Glass. Its walls are covered in photographs from after the war, all taken here on Poisonwood: freed slaves boarding Navy ships to leave the planet that brought them so much pain; Navy officers disembarking to retire on the world that they liberated; friends gone away; friends left behind. There are newer pictures, too: a Cruiser and two Falcons, side by side; armored troops walking away from a factory; a familiar captain talking to a man you know as Commander Nguyen. It is a strange feeling to be shown among heroes of a history that you've had only the smallest effect on.`
```
Comment (Anarchist2) [inline data/human/campaign events.txt:1792]: This paragraph is based off of the pre-war Poisonwood spaceport description. If you're changing the wording of this description, then I don't see any reason why you wouldn't also change that.
https://github.com/endless-sky/endless-sky/blob/c591b86cda71d35ee1b2fb08104ea462ca801d76/data/map%20planets.txt#L3953

Comment (ImprovisedUsername) [reply data/human/campaign events.txt:1792]: I didn't notice this description when I made the edit. My bad. It's updated now.

File: data/human/campaign events.txt [hunk 2]
```diff
@@ -1148,7 +1148,7 @@ event "oathkeepers founded"
 		fleet "Derelict Pirate" 10000
 	planet "Farpoint"
 		description `Farpoint, on the northernmost end of human space, is a rocky desert planet home to a Republic Navy base and not much else. Farming is possible only within greenhouses, and this system is too far from the galactic center to be desirable for trade or industry.`
-		description `	Lately the Navy base on Farpoint is growing, because it has become the new home of the Navy Oathkeepers, a small branch of the Navy that has sworn to stay uninvolved in the conflict with the Free Worlds.`
+		description `	Lately, the Navy base on Farpoint has been growing; it has become the new home of the Navy Oathkeepers, a small branch of the Navy that has sworn to stay uninvolved in the conflict with the Free Worlds.`
```
Comment (mOctave) [inline data/human/campaign events.txt:1151]:
```suggestion
		description `	Lately, the Navy base on Farpoint has been growing. It has become the new home of the Navy Oathkeepers, a small branch of the Navy that has sworn to stay uninvolved in the conflict with the Free Worlds.`
```

Probably just a period is better, because this sentence is kind of long.

File: data/human/campaign events.txt [hunk 3]
```diff
@@ -1251,10 +1251,10 @@ event "Thule becomes independent"
 		fleet "Large Free Worlds" 1500
 	planet "Thule"
 		add attributes "ex-pirate"
-		description `Thule is a mountainous world with a population of nearly a billion, settled in the early days of space exploration. When the colony was first established, the Earth government did not yet have the strength to control any system but its own, and when the Republic was formed, Thule did not join, and has repulsed all efforts to force them to do so.`
+		description `Thule is a mountainous world with a population of nearly a billion, settled in the early days of space exploration. When the colony was first established, the government on Earth did not yet have the strength to control any system but its own, and when the Republic was formed, Thule did not join, and has repulsed all efforts to force them to do so.`
```
Comment (mOctave) [inline data/human/campaign events.txt:1254]: Hmmm, the original here was a bit wordy, and the edits don't help much. Maybe:
```suggestion
		description `Thule is a mountainous world with a population of nearly a billion, settled in the early days of space exploration. When the colony was first established, the government on Earth did not yet have the strength to control any system but its own. Likewise, Thule refused to join the Republic when it was formed and has repulsed all efforts to force them to do so.`
```

File: data/human/campaign events.txt [hunk 4]
```diff
@@ -1753,7 +1753,7 @@ event "fw northern expansion"
 
 event "fw prison on Clink"
 	planet "Clink"
-		add description `Recently, the Free Worlds have converted Clink into a prison colony for Navy officers and support staff who were captured in the battle for the Rastaban system.`
+		add description `Recently, the Free Worlds has converted Clink into a prison colony for Navy officers and support staff who were captured in the battle for the Rastaban system.`
```
Comment (mOctave) [inline data/human/campaign events.txt:1756]: `have` is actually more widely used in the game, and I think it describes the federate aspect of the FW better than `has` would.

```suggestion
		add description `Recently, the Free Worlds have converted Clink into a prison colony for Navy officers and support staff who were captured in the battle for the Rastaban system.`
```

**Review by mOctave — COMMENTED** [review]
Looks good! Just a few tweaks.

File: data/human/campaign events.txt (hunk 3 above)
Comment (ImprovisedUsername) [reply data/human/campaign events.txt:1254]: I think "likewise" implies a similar action more than a consequence. "As such" may be better:

`As such, Thule refused to join the Republic when it was formed, and has since repulsed all efforts to force them to do so.`

**Review by Anarchist2 — COMMENTED** [review]
The pre-event descriptions of Martini, Thule and Smuggler's Den need to be updated to match the changes made here.

File: data/human/campaign events.txt [hunk 5]
```diff
@@ -1251,10 +1251,10 @@ event "Thule becomes independent"
 		fleet "Large Free Worlds" 1500
 	planet "Thule"
 		add attributes "ex-pirate"
-		description `Thule is a mountainous world with a population of nearly a billion, settled in the early days of space exploration. When the colony was first established, the Earth government did not yet have the strength to control any system but its own, and when the Republic was formed, Thule did not join, and has repulsed all efforts to force them to do so.`
+		description `Thule is a mountainous world with a population of nearly a billion, settled in the early days of space exploration. When the colony was first established, the government on Earth did not yet have the strength to control any system but its own. As such, Thule refused to join the Republic when it was formed, and has since repulsed all efforts to force them to do so.`
```
Comment (TheGiraffe3) [inline data/human/campaign events.txt:1254]:
```suggestion
		description `Thule is a mountainous world with a population of nearly a billion, settled in the early days of space exploration. When the colony was first established, the government on Earth did not yet have the strength to control any system but its own. As such, Thule refused to join the Republic when it was formed, and has since repulsed all efforts to force it to do so.`
```

File: data/map planets.txt [hunk 6]
```diff
@@ -5102,7 +5102,7 @@ planet Thshybothro
...
+	description `Thule is a mountainous world with a population of nearly a billion, settled in the early days of space exploration. When the colony was first established, the Earth government did not yet have the strength to control any system but its own. As such, Thule refused to join the Republic when it was formed, and has since repulsed all efforts to force them to do so. It is well-known that the planet sponsors pirate fleets which bring it cheap materials to bolster its economy, but the planet is so far from the center of Republic space that fighting a war here would be prohibitively expensive, and so they remain unopposed.`
```
Comment (TheGiraffe3) [inline data/map planets.txt:5105]:
```suggestion
	description `Thule is a mountainous world with a population of nearly a billion, settled in the early days of space exploration. When the colony was first established, the Earth government did not yet have the strength to control any system but its own. As such, Thule refused to join the Republic when it was formed, and has since repulsed all efforts to force it to do so. It is well-known that the planet sponsors pirate fleets which bring it cheap materials to bolster its economy, but the planet is so far from the center of Republic space that fighting a war here would be prohibitively expensive, and so they remain unopposed.`
```

**Review by TheGiraffe3 — APPROVED** [review]
